TelCo.ColorCoder/Utils/ColorPair.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Drawing;
+
+namespace TelCo.ColorCoder.Utils
```

Comment:
Name space could have been divided as data in TelCo.ColorCoder and functionaluty in .Util namespace

Reply:
Added Data namespace.